index.html

```diff
@@ -0,0 +1,174 @@
+      </header>
+      <section class="hero-main">
+        <div class="hero">
+          <p class="sub-heading">"LEADING GLOBAL TECH COMMUNITIES INTO AFRICA"</p>
+          <div class="h1-bg">
+            <h1 class="hero-h1">
+              GITEX Africa <span>Ethiopia Tour</span>
+              <span>2023</span>
+            </h1>
+          </div>
+          <div class="welcome">
+            <p class="welcome-text">
+              GITEX AFRICA, the continent’s largest all-inclusive tech event, will connect tech titans, governments, SMEs, start-ups, coders, investors and academia, to accelerate, collaborate and explore new ventures.
+            <br><br>
+              
+              Addis Ababa on September 2023.
+            </p>
+          </div>
+
+          <div class="date-location">
+            <h2>2023.02.30(WED) ~ 15(FRI)</h2>
+              <h3>@ millnium Hall , Addis Ababa</h3>
+          </div>
+        </div>
+        <img
+          class="friends"
+          src="assets/images/conferencethemenewImg.jpg"
+          alt="gitex conference theme."
+        />
+      </section>
+    </section>
+    <!-- Mid Section -->
+    <section class="main-program" id="program">
+      <h2>Main Events</h2>
+      <ul class="events">
+        <li class="event">
+          <img src="assets/images/program_icon_01 1.png" alt="keynote icon" />
+          <p class="event-title">Lecture</p>
+          <p class="event-desc">
+            Listen to the wizards from various continents about the ways of positive impacting 
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/jam.png" alt="" />
+          <p class="event-title">Workshop</p>
+          <p class="event-desc">
+            Try creating your own magic using open source power rather than just watching at it.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/expo.svg" alt="Expo Icon" />
+          <p class="event-title">Expo</p>
+          <p class="event-desc">
+            A tour of the studio and showcase of unique items that make the show
+            unforgetable.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/forum.svg" alt="" />
+          <p class="event-title">Forum</p>
+          <p class="event-desc">
+            Have the time to share your opinions with magicians for each topic
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/party.svg" alt="Party Icon" />
+          <p class="event-title">GetTog</p>
+          <p class="event-desc">
+            After the program, its gonna be get together and networking time
+            you dont wann miss it.
+          </p>
+        </li>
+      </ul>
+      <button class="get-ticket" type="button">Get your ticket Here</button>
+    </section>
+
+    <section class="guests" id="guests">
+      <h2>SPEAKERS</h2>
+      <div class="guests-list"></div>
+    </section>
```

Comment:
[COMPLIMENT] Great work on the ``More`` button in the **speakers** ``section`` 💯 💯 🔥